src/VignetteNodeSpec.ts

```diff
@@ -66,7 +66,7 @@ export const VignetteTableCellNodeSpec = (nodespec: NodeSpec): NodeSpec => ({
       base[1].style
     ) {
       base[1].style +=
-        'border-radius: 10px; border-style: solid; border-width: thin';
+      'border-radius: 10px; border:none!important; box-shadow: inset 0 0 0 1px #36598d;';
```

Comment:
Why the !important? Inline styles already have priority.

Reply:
Ryan,
It's necessary to use the !important declaration here because without it, the style will impact the inner borders of the ProseMirror table, resulting in doubled line thickness inside the table.